pynest/examples/sir_neuron.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+#
+# sir_neuron.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+sir_neuron example
+----------------------------------
+
+This scripts simulates a number of all-to-all connected sir neurons. One neuron
+is infected at the beginning of the simulation, all other neurons are
+susceptible. This example can be extended to the `sirs_neuron` or `sis_neuron` type by changing
+the `neuron_type` variable.
+"""
+
+import matplotlib.pyplot as plt
+import nest
+import numpy as np
+
+# create neuron population
+num_neurons = 100
+neuron_type = "sir_neuron"  # must be one of "sir_neuron" | "sirs_neuron" | "sis_neuron"
+sir_neurons = nest.Create(neuron_type, num_neurons)
+sir_neurons.beta_sir = 0.01  # downscale infectivity from default 0.1
+
+# connect sir_neurons all-to-all
```

Comment:
Why is the all-to-all connection parameter not used? https://nest-simulator.readthedocs.io/en/latest/synapses/connectivity_concepts.html#all-to-all